src/Context/GetoptParser.php

```diff
@@ -265,12 +270,14 @@ protected function splitLongOptionInput($input)
      * @return bool
      *
      */
-    protected function longOptionRequiresValue($option, $value, $name)
+    protected function longOptionRequiresValue($option, $value, $name=null)
     {
         if ($option->param == 'required' && trim($value) === '') {
-            $this->errors[] = new Exception\OptionParamRequired(
-                "The option '$name' requires a parameter."
-            );
+            if ($name) {
```

Comment:
In my humble opinion, explicit value checks are more desirable than relying on the truthyness of non-null values.  I'd write the check something like:  `if ($name !== null)`